assets/css/custom.scss

```diff
@@ -2167,7 +2171,7 @@ footer {
     }
   }
 }
-#masterpoint-blueprint-framework {
+#team-training {
   padding: 7.3rem 0 7.8rem;
   color: #fff;
   background: #031e1f;
```

Comment:
```suggestion
  background: #031e1f;
  h3 {
    color: #fff
  }
```
This section's background defaults to the green but is hard to see with the background:
![image](https://github.com/masterpointio/masterpoint.io/assets/35899715/8fe011e6-c530-4ef6-b57c-0b9842ce97a3)
The other section sets it to white using this same selector
![image](https://github.com/masterpointio/masterpoint.io/assets/35899715/7e7290d1-c6f7-4815-8e47-0a2171c86a76)

Reply:
Great find @oycyc -- Thanks for this. Will be pushed shortly!